diff --git a/src/Config.zig b/src/Config.zig
index 0bb03ab..5e8f44c 100644
--- a/src/Config.zig
+++ b/src/Config.zig
@@ -1627,16 +1627,12 @@ fn userAgentValidator(allocator: Allocator, args: *std.process.Args.Iterator, ua
     ua.* = try allocator.dupe(u8, str);
 }
 
-pub fn validateUserAgent(ua: []const u8) !void {
+pub fn validateUserAgent(ua: []const u8) error{ NonPrintable, Reserved }!void {
     for (ua) |c| {
         if (!std.ascii.isPrint(c)) {
             return error.NonPrintable;
         }
     }
-
-    if (std.ascii.indexOfIgnoreCase(ua, "mozilla") != null) {
-        return error.Reserved;
-    }
 }
 
 fn localeValidator(allocator: Allocator, args: *std.process.Args.Iterator, field: *[:0]const u8) !void {
diff --git a/src/network/HttpClient.zig b/src/network/HttpClient.zig
index 760f482..afd374e 100644
--- a/src/network/HttpClient.zig
+++ b/src/network/HttpClient.zig
@@ -468,8 +468,8 @@ pub fn getLanguages(self: *const Client) []const []const u8 {
 pub fn baselineHeaders(self: *const Client) [4]Transfer.RequestHeader {
     return .{
         .{ .name = "User-Agent", .value = self.getUserAgent() },
-        .{ .name = "Sec-Ch-Ua", .value = lp.Config.HttpHeaders.sec_ch_ua, .source = .fixed },
-        .{ .name = "Sec-Ch-Ua-Full-Version-List", .value = lp.Config.HttpHeaders.sec_ch_ua_full_version_list, .source = .fixed },
+        .{ .name = "X-ec-H-UA", .value = lp.Config.HttpHeaders.sec_ch_ua, .source = .fixed },
+        .{ .name = "X-ec-H-UA-Full-Version-List", .value = lp.Config.HttpHeaders.sec_ch_ua_full_version_list, .source = .fixed },
         // Omitting Accept-Language triggers bot-protection on some CDNs
         // (Akamai) when Accept-Encoding is present.
         .{ .name = "Accept-Language", .value = self.getAcceptLanguage() },
