diff --git a/node_modules/winston/lib/winston/logger.js b/node_modules/winston/lib/winston/logger.js
index 89dd4ac..612f235 100644
--- a/node_modules/winston/lib/winston/logger.js
+++ b/node_modules/winston/lib/winston/logger.js
@@ -306,6 +306,7 @@ class Logger extends Transform {
     } catch (ex) {
       throw ex;
     } finally {
+      this._writableState.sync = false;
       // eslint-disable-next-line callback-return
       callback();
     }
