diff --git a/node_modules/winston-transport/index.js b/node_modules/winston-transport/index.js
index a01b9b5..1545b2d 100644
--- a/node_modules/winston-transport/index.js
+++ b/node_modules/winston-transport/index.js
@@ -102,7 +102,7 @@ TransportStream.prototype._write = function _write(info, enc, callback) {
 
     return this.log(transformed, callback);
   }
-
+  this._writableState.sync = false;
   return callback(null);
 };
 
